VEDA/settings.py

```diff
@@ -110,6 +110,15 @@
 # Python dotted path to the WSGI application used by Django's runserver.
```

Comment:
We don't use `settings.py` in any of our IDAs. It probably shouldn't be used here, either. We generally break settings into sub-modules : base, local, devstack, test, production. See https://github.com/edx/cookiecutter-django-ida/tree/master/%7B%7Bcookiecutter.repo_name%7D%7D/%7B%7Bcookiecutter.repo_name%7D%7D/settings.

Reply:
You are right @clintonb - We will eventually be dividing the settings into sub parts. This is definitely on TODO list in our future work - CC @muzaffaryousaf